bundler.d/windows.rb

```diff
@@ -1,5 +1,5 @@
 group :windows do
   gem 'highline', :platform => :mingw
   gem 'win32-service', :platforms => :mingw
-  gem "win32-open3", :platforms => :mingw_18
+  gem 'wopen3', '>= 0.3', :platforms => :mingw
```

Comment:
Right, this gem is included and bundler will know about it and not install it. I was unsure and kept it as it does not any harm. You think I should remove it?